web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@momentum-ui/web-components",
-  "version": "2.9.1",
+  "version": "2.9.2",
```

Comment:
Do we need to increase the version in this PR?

Reply:
yea, this is what was finalised. with every momentum PR we will increase a minor version and publish it.